tests/integ/utils/sql_counter.py

```diff
@@ -419,7 +419,7 @@ def get_readable_sql_count_values(tr):
 
 
 def update_test_code_with_sql_counts(
-    sql_count_records: dict[str, dict[str, list[dict[str, Optional[Scalar]]]]]
+    sql_count_records: Dict[str, Dict[str, List[Dict[str, Optional[Scalar]]]]]
```

Comment:
im not sure if this change is actually needed.

Reply:
yes, that is needed, otherwise it will give an error "type" is not subscritble, it is because the python version restriction, snowpark pandas requires python 3.9 +, but snowpark python still allows 3.8